Core/src/main/java/com/plotsquared/core/queue/ChunkCoordinator.java

```diff
@@ -25,26 +25,23 @@
  */
 package com.plotsquared.core.queue;
 
-import com.sk89q.worldedit.world.biome.BiomeType;
-import com.sk89q.worldedit.world.block.BaseBlock;
-
-public class OffsetLocalBlockQueue extends DelegateLocalBlockQueue {
-    private final int ox;
-    private final int oy;
-    private final int oz;
-
-    public OffsetLocalBlockQueue(LocalBlockQueue parent, int ox, int oy, int oz) {
-        super(parent);
-        this.ox = ox;
-        this.oy = oy;
-        this.oz = oz;
-    }
+import com.plotsquared.core.util.task.PlotSquaredTask;
+
+public abstract class ChunkCoordinator implements PlotSquaredTask {
```

Comment:
This class needs documentation, as it's the uppermost one. Same with all the methods, as this is part of the API